docs/snippets/all/annotation_context_rects.cpp

```diff
@@ -23,7 +23,4 @@ int main() {
             {{3.0f, 3.0f}, {2.0f, 2.0f}}
         ).with_class_ids({1, 2})
     );
-
-    // Log an extra rect to set the view bounds
-    rec.log("bounds", rerun::Boxes2D::from_half_sizes({{2.5f, 2.5f}}));
```

Comment:
I reckon we should leave todo notes here pointing at the respective tickets to implement blueprint for cpp/rust

Reply:
This example didn't need the bounds. The ones that do need it, I did leave a TODO